### ISSUE TITLE
Focus indicators not visible enough

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1469]
## Description of issue

The custom focus indicator on the Team members page may not be visible to some users. The indicator has a contrast ratio of 1.64:1 with the surrounding white background and 1.24:1 with the grey button.

Our focus indicators are not all set up systematically. There are a lot of on-offs styling applying focus rings, backgrounds or shadows. This makes it hard to iterate on a11y and UX issues like this one. 

## Acceptance criteria
- [ ] Focus colours are part of a system and systematically applied acrosss the app
- [ ] We explore theming GCDS components
- [ ] We design rules for interactive elements that should receive focus. This will guide us to apply styling that is predictable and perceivable to all users. 

## Potential fix

- Design more contrasty focus indicators. 
- Implement all at once, rather than slowly rolling out. Focus indicators should be predictable. 

## Resources
- [WCAG 2.2 focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

**Comment by yaelberger-commits** [comment]
Could look to match GCDS colour

**Comment by yaelberger-commits** [comment]
Also ready to demo soon

**Comment by amazingphilippe** [comment]
Worked on a proof of concept to hook into the GCDS CSS property API. We can set any focus colour we want, not just the ones GCDS sets by default.

**Comment by yaelberger-commits** [comment]
In a review app

**Comment by amazingphilippe** [comment]
Branch with the review app: https://github.com/cds-snc/notification-admin/pull/2052
